### DISCUSSION TITLE
Implement a possible `check` function to determine whether or not `mkcert` has been run before

### DISCUSSION THREAD

**Discussion by echoboomer** [FiloSottile/mkcert#516] [Ideas]
It would be great if there were some sort of `-check` flag to return whether or not a system/other trust store has been created already. In cases where `mkcert` is a dependency of some other platform or service, advising users to stop and run `mkcert -install` if the trust store either does or does not exist would be ideal without needing to run `-install`.

**Comment by harelzar** [comment]
mkcert --version
